mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/Dsl.kt

```diff
@@ -81,8 +83,11 @@ data class TestEnvironment(
  * information.
  */
 suspend fun ComposeContentTestRule.setContentWithTestEnvironment(
-    store: Store = emptyStore(),
-    auth: Auth = emptyAuth(),
+    userId: String = DefaultId,
```

Comment:
Missing documentation for the parameter.

```kotlin
* @param userId the user identifier for the test environment user.
```

Reply:
Fixed in https://github.com/epfl-SDP/android/pull/390/commits/c4590eaf815cf86e4d3587ab8be2e0e7e6cc2ede